build/dependencies.props

```diff
@@ -75,7 +75,7 @@
     <MicrosoftExtensionsLocalizationAbstractionsPackageVersion>2.2.0</MicrosoftExtensionsLocalizationAbstractionsPackageVersion>
     <MicrosoftExtensionsLocalizationPackageVersion>2.2.0</MicrosoftExtensionsLocalizationPackageVersion>
     <MicrosoftExtensionsLoggingAbstractionsPackageVersion>2.2.0</MicrosoftExtensionsLoggingAbstractionsPackageVersion>
-    <MicrosoftExtensionsLoggingAzureAppServicesPackageVersion>2.2.0</MicrosoftExtensionsLoggingAzureAppServicesPackageVersion>
+    <MicrosoftExtensionsLoggingAzureAppServicesPackageVersion>2.2.5-servicing-19212-04</MicrosoftExtensionsLoggingAzureAppServicesPackageVersion>
```

Comment:
To make this work with ProdCon, you'll need to also move this up into the first property group in this file https://github.com/aspnet/AspNetCore/blob/fc7ef32c73089c7db338fa0acd88c86fda7712a3/build/dependencies.props#L2-L3

Reply:
Can you remove this line here now that it's been copied into the section above?

Reply:
Incidentally, this change will fix https://github.com/aspnet/AspNetCore-Internal/issues/2252